docs: #377 update mdbook readme (#1118)
mozilla/rust-code-analysis: rust-code-analysis-book/src/commands/README.md

# Commands

> The **rust-code-analysis-cli** offers a range of **commands** to analyze source code and extract valuable insights. Each command **may** include parameters specific to the task it performs. Below, we describe the core types of commands available in **rust-code-analysis-cli**.

Reviewer: ```suggestion
**rust-code-analysis-cli** offers a range of **commands** to analyze and extract information from source code. Each command **may** include parameters specific to the task it performs. Below, we describe the core types of commands available in **rust-code-analysis-cli**.
```

## REST API

> The **rust-code-analysis-web** can function as a server using a REST API. This allows users to send source code via HTTP and receive corresponding metrics in `JSON` format, enabling easy integration into web applications or other services.

Reviewer: ```suggestion
**rust-code-analysis-web** runs a server offering a REST API. This allows users to send source code via HTTP and receive corresponding metrics in `JSON` format.
```
